patches/0034-Disable-wallpapers.patch

```diff
@@ -0,0 +1,38 @@
+From f063b334e2728a640c92b86a67d5921b8cc9be5e Mon Sep 17 00:00:00 2001
```

Comment:
Somewhere earlier, a call to download wallpapers was added in "patches/0011-Disabling-Telemetry-CrashReports.patch". Here, wallpapers are disabled. Why was the download wallpapers needed?